ledger/src/shred.rs

```diff
@@ -1267,6 +1283,10 @@ pub fn verify_test_data_shred(
     }
 }
 
+pub fn should_chain_merkle_shreds(_slot: Slot, cluster_type: ClusterType) -> bool {
```

Comment:
moved from `standard_broadcast_run` in order to resolve dependencies.

Reply:
just a note to move this back if the metric gate on `should_chain_merkle_shreds` is removed.